README.md

```diff
@@ -197,7 +197,9 @@ helmfiles:
   # Override state values
   values:
   # Values files merged into the nested state's values
-  - additiona.values.yaml
+  - additional.values.yaml
+  # One important aspect of using values here is that they first need to be defined in the releases section
```

Comment:
```suggestion
  # One important aspect of using values here is that they first need to be defined in the values section
```

Reply:
I'd suggest this to clarify more that this `values` does nothing to do with `releases[].values`.